app/model/persisted/eventschedules/SkillType.scala

```diff
@@ -16,12 +16,13 @@
 
 package model.persisted.eventschedules
 
-import model.persisted.eventschedules.SkillType.SkillType
 import play.api.libs.json.{ Format, JsString, JsSuccess, JsValue }
 import reactivemongo.bson.{ BSON, BSONHandler, BSONString }
 
 import scala.language.implicitConversions
 
+
+@deprecated("We should get rid of this enum and change code accordingly to use AssessorSkill, then rename AssessorSkill to SkillType")
```

Comment:
This needs a date to be a valid deprecated statement (deprecated with no date is ironically deprecated). I'd also discourage us from doing this and say you should fix it/change it rather than deprecate it - it's just going to swell our warnings